### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by RiccardoCherchi** [vitalik/django-ninja#511]
I had trouble to use Ninja Forms, the form is not working with the PUT method, but only POST

```py
@router.post("/profile/edit", response={200: dict, 400: dict})
def edit_profile(request: Request, profile: EditProfileSchema = Form(...)):
    print(profile.dict())
```

result:
`{'username': 'test', 'fullname': 'test', 'delete_image': True, 'bio': '', 'size': ''}`

this code works only if i leave the post method, if put is set this is te result:

`{'username': None, 'fullname': None, 'delete_image': None, 'bio': None, 'size': None}`

Versions:
 - Python version: 3.10.3
 - Django version: 4.0.6
 - Django-Ninja version: 0.19.1

**Comment by vitalik** [comment]
@RiccardoCherchi 

well this is general django approeach for POST data... basically put and patch methods are not collecting form data into request.POST

you can fix that by adding middleware like here - https://github.com/vitalik/django-ninja/issues/417#issuecomment-1092545699

**Comment by bimbims125** [comment]
> @RiccardoCherchi
> 
> well this is general django approeach for POST data... basically put and patch methods are not collecting form data into request.POST
> 
> you can fix that by adding middleware like here - [#417 (comment)](https://github.com/vitalik/django-ninja/issues/417#issuecomment-1092545699)

where i can put the middleware code?
